src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -25,6 +27,12 @@ class UseCaseID(Enum):
     UseCaseKey.PERFORMANCE: UseCaseID.TRANSACTIONS,
 }
 
+USE_CASE_ID_WRITES_LIMIT_QUOTA_OPTION_NAME = {
```

Comment:
nit can we rename this to `USE_CASE_ID_WRITES_LIMIT_QUOTA_OPTIONS`

Reply:
Lol I was just commenting on your pr